src/components/message/MessageInput.tsx

```diff
@@ -1,108 +1,98 @@
 import { Box } from "@twilio-paste/core";
-import { useState, useLayoutEffect, KeyboardEventHandler } from "react";
+import { ChatComposer } from "@twilio-paste/core/chat-composer";
+import {
+  $getRoot,
+  ClearEditorPlugin,
+  CLEAR_EDITOR_COMMAND,
+  COMMAND_PRIORITY_LOW,
+  EditorState,
+  KEY_ENTER_COMMAND,
+  useLexicalComposerContext,
+} from "@twilio-paste/lexical-library";
+import { useEffect } from "react";
 import MessageFile from "./MessageFile";
 
 interface MessageInputProps {
   message: string;
   onChange: (message: string) => void;
-  onKeyPress: KeyboardEventHandler<HTMLInputElement>;
+  onEnterKeyPress: () => void;
   onFileRemove: (file: string) => void;
   assets: File[];
 }
 
-function useWindowSize() {
-  const [size, setSize] = useState(0);
-  useLayoutEffect(() => {
-    function updateSize() {
-      setSize(window.innerWidth);
-    }
-    window.addEventListener("resize", updateSize);
-    updateSize();
-    return () => window.removeEventListener("resize", updateSize);
-  }, []);
-  return size;
```

Comment:
All these custom functions for determining width and gracefully fitting the content to a smaller screen has been removed (most of the red parts of the diff in this PR) as the ChatComposer component just handles the size adjustments automatically.